core/src/main/java/org/apache/accumulo/core/clientImpl/CompressedIterators.java

```diff
@@ -70,10 +69,13 @@ public ByteBuffer compress(IteratorSetting[] iterators) {
       Map<String,String> opts = is.getOptions();
       out.writeVInt(opts.size());
 
-      for (Entry<String,String> entry : opts.entrySet()) {
-        out.writeVInt(getSymbolID(entry.getKey()));
-        out.writeVInt(getSymbolID(entry.getValue()));
+      if (!opts.isEmpty()) {
+        opts.forEach((k, v) -> {
```

Comment:
I do like this tuple view of Entry rather than iterating over the entrySet, and having to call getKey and getValue later.